test/types/function-type.spec.js

```diff
@@ -44,120 +44,123 @@ describe('function type', () => {
     expect(function() {}, 'to inspect as', 'function () {}');
   });
 
-  it('should inspect a one-line function correctly', () => {
-    /* eslint-disable no-unused-vars */
-    expect(
-      // prettier-ignore
-      function() { var a = 123;a = 456; },
-      'to inspect as',
-      'function () { var a = 123;a = 456; }'
-    );
-    /* eslint-enable no-unused-vars */
-  });
+  const isDeno = typeof Deno !== 'undefined';
+  if (!isDeno) {
+    it('should inspect a one-line function correctly', () => {
+      /* eslint-disable no-unused-vars */
+      expect(
+        // prettier-ignore
+        function() { var a = 123;a = 456; },
```

Comment:
Seems like the reason why these tests don't pass in deno is that the transpilation of the tests realizes that the code inside the function can be optimized away. Could you try changing it to something with side effects, such as the below?

```suggestion
        function() { var a = 123;alert(a); },
```

... or find a way to make the transpilation leave it alone?

Reply:
Hmm interesting - the symptoms I saw was when this was inspected it came out as “native code” so I’d assumed it was a bug in Deno. I’d much rather these ran too so will see why I can do!